apps/apps.go

```diff
@@ -0,0 +1,161 @@
+package apps
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+
+	client "github.com/deis/controller-sdk-go"
+	"github.com/deis/controller-sdk-go/api"
+)
+
+const workflowURLPrefix = "deis."
+
+// List lists apps on a Deis controller.
+func List(c *client.Client, results int) ([]api.App, int, error) {
+	body, count, err := c.LimitedRequest("/v2/apps/", results)
```

Comment:
@Joshua-Anderson this is probably a huge refactor, so just throwing it out there as an idea - would it be possible to have `LimitedRequest` return an `io.ReadCloser` instead of the `string` it returns now?